components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/internal/BundleActivator.java

```diff
@@ -63,8 +63,16 @@
     protected void start(BundleContext bundleContext) {
         try {
             Context ctx = jndiContextManager.newInitialContext();
-            DataSource dataSource = new DataSourceImpl((HikariDataSource) ctx.lookup("java:comp/env/jdbc/WSO2AMDB"));
-            DAOUtil.initialize(dataSource);
+            DataSource dataSourceAMDB = new DataSourceImpl(
+                    (HikariDataSource) ctx.lookup("java:comp/env/jdbc/WSO2AMDB"));
```

Comment:
since we are using it only in here, do we need to move to a constant? AM_DB currently uses the string literal inline. If we are moving to constants we have to move both.